Modules/ossaudiodev.c

```diff
@@ -542,9 +542,7 @@ oss_self(PyObject *self, PyObject *unused)
 static PyObject *
 oss_exit(PyObject *self, PyObject *unused)
 {
-    _Py_IDENTIFIER(close);
```

Comment:
We also need to remove `#define NEEDS_PY_IDENTIFIER`, right?

Reply:
done